shared/src/main/scala/pl/metastack/metarx/Dep.scala

```diff
@@ -0,0 +1,19 @@
+package pl.metastack.metarx
+
+/** @see [[Sub.dep()]] */
+private[metarx] class Dep[T, U](sub: Sub[T],
+                                fwd: ReadChannel[T] => ReadChannel[U],
+                                bwd: ReadChannel[U] => ReadChannel[T]) {
+  private var value: U = _
```

Comment:
Technically, do we need this?  Can't we actually just have `def get: U = internal.get`?